src/chatd.cpp

```diff
@@ -4039,148 +4064,153 @@ void Chat::onMsgUpdated(Message* cipherMsg)
     })
     .then([this, updateTs, richLinkRemoved](Message* msg)
     {
-        assert(!msg->isPendingToDecrypt()); //either decrypted or error
```

Comment:
All this `.then()` block has been moved, without changes, to `onMsgUpdatedAfterDecrypt()`, so it can be called directly for imported messages.